ONSdigital/eq-survey-runner
# Add security headers (copy of #163)

**_Changes**_
Pins all pypi dependencies to SHA256 hashes to stop downstream tampering

**_How to test**_
Run a clean install using pip install -U -I -r requirements.txt
Edit one of the hashes in requirements.txt
Re-run the command pip install -U -I -r requirements.txt
It should fail, warning of a tampering with one of the packages.

**_Who can test**_

Anyone but @dhilton
